monitor/test_result.py

```diff
@@ -32,22 +38,30 @@ def create_test_result_from_dict(test_result_dict: dict) -> Optional['TestResult
             else:
                 return ElementaryTestResult(**test_result_dict)
         except Exception:
-            logger.error(f"Failed parsing test result - {json.dumps(test_result_dict)}")
+            logger.exception(f"Failed parsing test result - {json.dumps(test_result_dict)}")
```

Comment:
Usually its a good practice, but in this case the user will see the error exception itself and might get confused.
This exception is not breaking the flow, it will send (in the future) an "error" alert with the other alerts.
So printing the trace of the error when a user runs the edr can create a feeling that he shouldn't trust the edr.